dashboard/app/models/concerns/serialized_properties.rb

```diff
@@ -40,6 +40,10 @@ def serialized_attrs(*args)
       (serialized_properties[self.to_s] ||= []).concat args
     end
 
+    def get_external_params
```

Comment:
we could just call this permitted_params. Maybe add a comment explaining why we need to do this instead of using the serialized_attrs thing we were doing before?